staging/src/k8s.io/kubectl/pkg/cmd/cp/cp_test.go

```diff
@@ -411,7 +412,7 @@ func TestTarUntarWrongPrefix(t *testing.T) {
 	if err := os.MkdirAll(path.Dir(filepath), 0755); err != nil {
 		t.Fatalf("unexpected error: %v", err)
 	}
-	createTmpFile(t, filepath, "sample data")
+	createTmpFile(t, filepath, "sample data", 0666)
```

Comment:
How about adding a const for the perm so you can check later in the code if the perm is correct?
Additional to that, I would go with 0664 even in tests. :)